engine/collection/compliance/core.go

```diff
@@ -89,7 +89,9 @@ func (c *Core) OnBlockProposal(originID flow.Identifier, proposal *messages.Clus
 		Uint64("block_view", header.View).
 		Hex("block_id", logging.Entity(header)).
```

Comment:
Since we are logging the `collection_id` and `block_id`, we can find what block is turned into a guarantee by following the `OnFinalizedBlock` log where it logs the finalized block id. Any collection_id associated with the finalized block id will be turned into a guarantee. Useful for debugging